app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/FeedFragment.java

```diff
@@ -71,6 +84,16 @@ public void onFailure(Call<ArrayList<NewsItem>> call, Throwable t) {
             }
         });
     }
+
+    public static FeedFragment newInstance(int someInt) {
+        FeedFragment myFragment = new FeedFragment();
+
+        Bundle args = new Bundle();
+        args.putInt("someInt", someInt);
```

Comment:
@Anna812 can we name this meaningful instead "someInt". eg: a boolean named onlyFavorites